public/docs/advanced/commands.md

```diff
@@ -1,129 +1,215 @@
+
+* `--network <name>`: Specify the network to use, using artifacts specific to that network. Network name must exist in the configuration.
+
+
+### help
+
+Display a list of all commands and then exit.
+
+```shell
+truffle help
+```
 
-* `--network name`: Specify the network to use, using artifacts specific to that network.
 
 ### init
 
-Create a completely new app within the current working directory. Will add default contracts, tests and frontend configuration.
+Initialize new Ethereum project with example contracts and tests.
 
-```none
-$ truffle init
+```shell
+truffle init
 ```
 
+Creates a new and empty Truffle project within the current working directory. Will add default contracts, tests and frontend configuration. Takes no arguments.
+
+<p class="alert alert-info">
+**Note**: Older versions of Truffle used `truffle init bare` to create an empty project. This usage has been deprecated. Those looking for the MetaCoin example that used to be available through `truffle init` should use `truffle unbox MetaCoin` instead.
+</p>
+
+
 ### install
 
 Install a package from the Ethereum Package Registry.
 
-```none
-$ truffle install <package name>@<version>
+```shell
+truffle install [package_name]<@version>
 ```
 
-The @`version` parameter syntax is optional. See the [Package Management with EthPM](/docs/getting_started/packages-ethpm) section for more details.
+Parameters:
+
+* `package_name`: Name of the package as listed in the Ethereum Package Registry.
+
+Optional parameters:
+
+* `<@version>`: When specified, will install a specific version of the package, otherwise will install the latest version.
+
+See the [Package Management with EthPM](/docs/getting_started/packages-ethpm) section for more details.
+
+
 
 ### migrate
 
-Run your project's migrations. See the [Migrations](/docs/getting_started/migrations) section for more details.
+Run migrations to deploy contracts.
 
-```none
-$ truffle migrate
+```shell
+truffle migrate [--reset] [-f <number>] [--network <name>] [--compile-all] [--verbose-rpc]
 ```
 
+Unless specified, this will run from the last completed migration. See the [Migrations](/docs/getting_started/migrations) section for more details.
+
 Optional parameters:
 
 * `--reset`: Run all migrations from the beginning, instead of running from the last completed migration.
-* `-f number`: Run contracts from a specific migration.
-* `--network name`: Specify the network to use, saving artifacts specific to that network.
-* `--compile-all`: Compile all contracts instead of intelligently choosing.
+* `-f <number>`: Run contracts from a specific migration. The number refers to the prefix of the migration file.
+* `--network <name>`: Specify the network to use, saving artifacts specific to that network. Network name must exist in the configuration.
+* `--compile-all`: Compile all contracts instead of intelligently choosing which contracts need to be compiled.
 * `--verbose-rpc`: Log communication between Truffle and the RPC.
```

Comment:
RPC is wrong here as well (this stuff is way old). I'd say `the Ethereum client` here.